src/Maker/MakeAuthenticator.php

```diff
@@ -192,13 +205,21 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
 
         // update security.yaml with guard config
         $securityYamlUpdated = false;
+
+        $entryPoint = $input->getOption('entry-point');
+
+        if ($this->useSecurity52 && self::AUTH_TYPE_FORM_LOGIN !== $input->getArgument('authenticator-type')) {
+            $entryPoint = false;
+        }
+
```

Comment:
Up in `interact()`, we still call `$interactiveSecurityHelper->guessEntryPoint()`, which still uses guard logic and may ask the user about the entry point interactively. I think we don't want that in the new system.

Reply:
done